.huskyrc.js

```diff
@@ -1 +0,0 @@
-module.exports = require('kcd-scripts/husky')
```

Comment:
This must be an ESM module with our configuration now. But I'm planning on updating the version of husky so I just removed it for now.